script/cms/test-bundle-transformers.js

```diff
@@ -66,9 +65,16 @@ const {
   print: printIndex,
   bundle,
   help,
-  transformUnpublished,
+  'transform-unpublished': transformUnpublished,
+  buildtype,
 } = commandLineArgs(optionDefinitions);
 
+const contentDir = path.resolve(
+  __dirname,
+  `../../.cache/${buildtype}/cms-export-content`,
```

Comment:
Moving forward does this mean the `buildtype` flag is required? I noticed when I exclude it from the command line I get an error because it's `undefined`.